packages/strapi-plugin-upload/admin/src/components/InputMedia/index.js

```diff
@@ -1,7 +1,9 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { get } from 'lodash';
 
-import { getTrad } from '../../utils';
+import { getTrad, prefixFileUrlWithBackendUrl, createFileToEdit } from '../../utils';
```

Comment:
After #5624 the `prefixFileUrlWithBackendUrl` will be located in the `strapi-helper-plugin` .
Also why didn't you use `formatFileForEditing` instead of `createFileToEdit`?